### ISSUE TITLE
PS4 won't turn off

### ISSUE THREAD

**Issue by samfadel** [GoldHEN/GoldHEN#224]
I have ps4 slim fw: 11.00 before update goldhen to ver 2.4b18 my ps4 shut down normal on ver 2.4b17.2 now I won't turn off

**Comment by Chbich62** [comment]
bug still present in rest mode and when turning off my ps4 pro 11.00

**Comment by samfadel** [comment]
2.4b17.2 no problem with rest mode or turning off

**Comment by Up-Zer0** [comment]
Same here on 2.4b.17.3 and 2.4b.18

**Comment by KIMO-in** [comment]
I already posted this issue before...but sistro responded to me saying "so don't use it" 😂

**Comment by SiSTR0** [comment]
> I already posted this issue before...but sistro closed it and responded to me saying "so don't use it" 😂

yes your issue was arrogant, so I repeat you feel free not to use it, if instead you want to be useful connect on discord and try to give some more info to the support. The invitation is obviously open to everyone.

**Comment by KIMO-in** [comment]
> > I already posted this issue before...but sistro closed it and responded to me saying "so don't use it" 😂
> 
> yes your issue was arrogant, so I repeat you feel free not to use it, if instead you want to be useful connect on discord and try to give some more info to the support. The invitation is obviously open to everyone.

No...bro i meant to feedback...i don't know why you thought i was arrogant 🤔

**Comment by SiSTR0** [comment]
Contact GoldHEN support on [discord](https://discord.com/invite/pR5NTEVBGt)

**Comment by Zoko59** [comment]
> Contact GoldHEN support on [discord](https://discord.com/invite/pR5NTEVBGt)

Thank you for your work, you have served millions of people and people forget that.

**Comment by IoanaAdrian** [comment]
I've managed to consistently replicate it. In my case it only happens when powering off immediately after powering on from rest mode: it is due to the fact that the Raspberry Pi has no time to detect that GoldHEN is already running after powering on from rest mode, so the Raspberry isn't triggered to shut down. This way, when the PS4 tries to shut down, the Raspberry is still sending PPPoE requests which triggers the kernel to panic, resulting in it to not power off.
To solve this, I just wait 1 minute or 2 for the Raspberry to detect that GoldenHEN is running after the PS4 recovers from rest mode, or I unplug the Raspberry from the USB port to force it to shut down(needs to be done before turning off the PS4).
This doesn't seem to be dependent on the GoldenHEN version at all.

My setup: PS4 pro CUH 7216b, 11.0, Raspberri Pi 0w running stooged/PI-Pwn
